operators/community-windows-machine-config-operator/5.1.1/manifests/community-windows-machine-config-operator.5.1.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,351 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
+    certified: "false"
+    createdAt: "2022-06-08T15:43:09Z"
+    description: An operator that enables Windows container workloads on OCP
+    olm.skipRange: '>=4.0.0 <5.0.0'
```

Comment:
this needs to be removed in favor of skipping the broken version

Reply:
Sorry, since both 5.0.0 and 5.1.0 have problems we should just increase this skiprange to include them 
https://github.com/openshift/windows-machine-config-operator/issues/1097

Reply:
That also means that the `replaces` should be removed